docs/ako_external_traffic_policy_local.md

```diff
@@ -0,0 +1,235 @@
+# ExternalTrafficPolicy Local Support in AKO
+
+This feature is supported starting from AKO version **1.14.1** and enables AKO to support the `externalTrafficPolicy` field set as `Local` in NodePort services. This ensures that only nodes with running application pods are added as pool servers in the Avi Controller, improving traffic efficiency and preserving client IP addresses.
+
+## Overview
+
+The `externalTrafficPolicy` field in Kubernetes services determines whether external traffic should be routed to node-local or cluster-wide endpoints. When set to `Local`, the kube-proxy only distributes load to pods running on the same node, which:
+
+- Preserves the client IP address
+- Ensures nodes without pods are not included in service endpoints
+- Reduces network hops
+
+## Problem Statement
+
+Prior to this feature, AKO did not respect the `externalTrafficPolicy: Local` setting and would:
+- Add all cluster nodes as pool servers regardless of pod distribution
+- Rely on health monitors to mark down nodes without running pods
+- Result in unnecessary resource overhead from monitoring nodes that should not receive traffic
+
+## Solution
+
+With this feature, AKO now:
+1. **Detects Local Traffic Policy**: When processing a NodePort service, AKO checks if `externalTrafficPolicy` is set to `Local`
+2. **Selective Pool Server Addition**: Only adds nodes with active running pods as pool servers in the Avi Controller.
+
+## Supported Platforms
+
+This feature works with:
+- **Kubernetes clusters** using Ingress resources
+- **Kubernetes and OpenShift clusters** using Gateway API (HTTPRoute resources)
```

Comment:
@DixitAakash Shouldn't this applicable to LB type services also with Nodeport mode?